### PR TITLE
Add scripts to deploy eth and ipfs infrastructure

### PR DESCRIPTION
**Pull request by jkilpatr**

### CODE REVIEW

**Comment by jkilpatr** [comment]
running at eth-infra.altheamesh.com please test.

File: scripts/roles/install-parity/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+parity_bin_url: https://releases.parity.io/v2.0.6/x86_64-unknown-linux-gnu/parity
+parity_websocket_port: 8546
+parity_peer_port: 30303
+parity_rpc_port: 8545
+parity_ipfs_port: 6001
```
Comment (sebohe) [inline scripts/roles/install-parity/defaults/main.yml]: :+1:

File: scripts/setup-infra-node.retry [hunk 2]
```diff
@@ -0,0 +1 @@
+206.189.199.0
```
Comment (sebohe) [inline scripts/setup-infra-node.retry]: Is this file needed?
